app/(backend)/dtos/aimodel.dto.ts

```
@@ -1,88 +1,89 @@
-    providerId: z
-      .string()
-      .uuid({ message: "Invalid provider ID format" }),
+  // Skema untuk identifier model
+  static readonly IDENTIFIER_SCHEMA = z
+    .string()
+    .min(1, { message: "Model identifier cannot be empty" })
+    .max(255, { message: "Model identifier is too long" })
+    .refine((val) => /^[a-z0-9\-\.]+$/.test(val), {
+      message: "Model identifier must contain only lowercase letters, numbers, hyphens, and dots"
+    });
+}
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)